deploy.sh

```
@@ -8,45 +8,62 @@ LARAVEL_DIR="laravel"
+echo "Step 2/11: Creating deployment directory..."
 mkdir -p $DEPLOY_DIR
 
 # Langkah 3: Salin file ke folder deployment
-echo "Step 3/10: Copying project files..."
+echo "Step 3/11: Copying project files..."
 rsync -a --exclude='vendor/' --exclude='node_modules/' --exclude='deploy_rumahweb/' ./ $DEPLOY_DIR/
 
 # Langkah 4: Pindah ke folder deployment
-echo "Step 4/10: Entering deployment directory..."
+echo "Step 4/11: Entering deployment directory..."
 cd $DEPLOY_DIR
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)